README.md

```diff
@@ -40,6 +60,14 @@ To change these settings, simply edit the `.env` file in the root directory of t
 - Either:
   - Docker and Docker Compose (preferred)
   - Go 1.21 or later
-- Local LLM server
+- Docker Model Runner (for AI model integration)
+
+## Docker Model Runner Setup
+
+This project uses Docker's first-class Compose integration with AI models. To run the projects locally, you need to enable Docker Model Runner:
+
+1. Follow the official documentation: https://docs.docker.com/ai/model-runner/#enable-dmr-in-docker-engine
+2. The run.sh script will automatically pull the required AI model (`ai/llama3.2:1B-Q8_0`)
```

Comment:
We can remove the pulling of models through the script, since this is now part of the Compose integration.